config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   get '/auth/slack/callback' => 'session#slack_callback'
 
-  root 'pages#index'
+  root 'pre_built_pages#top'
+  get '/wip_term' => 'pre_built_pages#wip_term'
+  get '/thesis' => 'pre_built_pages#thesis'
+
   scope :pages do
+    get   '/' => 'pages#index', as: :pages
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.